### PR TITLE
PT-417 | Make min_group_size and max_group_size nullable

### PR DESCRIPTION
**Pull request by quyenlq**

### CODE REVIEW

File: occurrences/models.py [hunk 1]
```diff
@@ -117,8 +117,12 @@ class Occurrence(TimestampedModel):
         blank=True,
         null=True,
     )
-    min_group_size = models.PositiveSmallIntegerField(verbose_name=_("min group size"))
-    max_group_size = models.PositiveSmallIntegerField(verbose_name=_("max group size"))
+    min_group_size = models.PositiveSmallIntegerField(
+        verbose_name=_("min group " "size"), blank=True, null=True
+    )
+    max_group_size = models.PositiveSmallIntegerField(
+        verbose_name=_("max group " "size"), blank=True, null=True
```
Comment (tuomas777) [inline occurrences/models.py:124]: I think I've seen this kind of strings before somewhere...

File: occurrences/schema.py [hunk 2]
```diff
@@ -406,8 +406,9 @@ def validate_enrolment(study_group, occurrence, new_enrolment=True):
     if occurrence.cancelled:
         raise EnrolCancelledOccurrenceError("Cannot enrol cancelled occurrence")
     if (
-        study_group.group_size > occurrence.max_group_size
-        or study_group.group_size < occurrence.min_group_size
+        occurrence.max_group_size and study_group.group_size > occurrence.max_group_size
```
Comment (tuomas777) [inline occurrences/schema.py:409]: After this change, if `max_group_size` is `0` the check will be skipped, which is probably not what you want.

Comment (quyenlq) [reply occurrences/schema.py]: Good point, but that'd be something we want. This validation should be run only if the field value > 0. Still made a change to improve readability

File: occurrences/schema.py [hunk 3]
```diff
@@ -406,8 +406,8 @@ def validate_enrolment(study_group, occurrence, new_enrolment=True):
     if occurrence.cancelled:
         raise EnrolCancelledOccurrenceError("Cannot enrol cancelled occurrence")
     if (
-        study_group.group_size > occurrence.max_group_size
-        or study_group.group_size < occurrence.min_group_size
+        study_group.group_size > occurrence.max_group_size > 0
+        or 0 < study_group.group_size < occurrence.min_group_size
```
Comment (tuomas777) [inline occurrences/schema.py:410]: Hmm, doesn't this now throw when `max_group_size` or `min_group_size` is `None` ?

Comment (quyenlq) [reply occurrences/schema.py]: Ahhh, my bad, must be to hurry for the lunch, reverted

**Review by tuomas777 — APPROVED** [review]
ok let's go! 🙂

**Pull request merged**